game.py

```diff
@@ -407,6 +412,53 @@ def on_update(self):
                     self.board.overlay.update_unit(self.players[0].units[new_idx])
                 self.map_pos = (clamp(self.board.selected_unit.location[0] - 12, -1, 77),
                                 clamp(self.board.selected_unit.location[1] - 11, -1, 69))
+        elif pyxel.btnp(pyxel.KEY_M):
+            if self.on_menu and self.menu.in_wiki and self.menu.wiki_showing is not None:
+                self.menu.wiki_showing = None
+            if self.on_menu and self.menu.in_game_setup:
+                self.menu.in_game_setup = False
+            if self.on_menu and self.menu.loading_game:
+                self.menu.loading_game = False
+            if self.game_started and self.board.overlay.is_elimination():
+                self.board.overlay.toggle_elimination(None)
+            elif self.game_started and self.board.overlay.is_night():
+                self.board.overlay.toggle_night(None)
+            elif self.game_started and self.board.overlay.is_close_to_vic():
+                self.board.overlay.toggle_close_to_vic([])
+            elif self.game_started and self.board.overlay.is_bless_notif():
+                self.board.overlay.toggle_blessing_notification(None)
+            elif self.game_started and self.board.overlay.is_constr_notif():
+                self.board.overlay.toggle_construction_notification(None)
+            elif self.game_started and self.board.overlay.is_lvl_notif():
+                self.board.overlay.toggle_level_up_notification(None)
+            elif self.game_started and self.board.overlay.is_controls():
+                self.board.overlay.toggle_controls()
+            elif self.game_started and self.board.overlay.is_investigation():
+                self.board.overlay.toggle_investigation(None)
```

Comment:
I believe everything above here isn't necessary, it'll give the M key far too much power.

Reply:
Addressed in 16213ad5288fc576e9fda877648ab6728966dae4